### ISSUE TITLE
type conversion error

### ISSUE THREAD

**Issue by jycor** [charles-001/dolphie#79]
I'm hitting this error:
```python
│   234 │   if replica_lag is not None:              
│   235 │   │   if data.get("SQL_Delay"):            
│ ❱ 236 │   │   │   replica_lag -= data["SQL_Delay"] 
│   237 │   │                                        
│   238 │   │   lag_color = "green"                  
│   239 │   │   if replica_lag >= 20:
...
TypeError: unsupported operand type(s) for -=: 'int' and 'str'
```
from [here](https://github.com/charles-001/dolphie/blob/353e65cc051d2b3ccb0ac8474b8abf3e9de33601/dolphie/Panels/replication_panel.py#L236).

Either an `int(...)` cast is necesary or 
```python
@dataclass
class Replica:
....
    replication_status: Dict[str, str] = field(default_factory=dict)
...
```
should be defined as `Dict[str, int]` from [here](https://github.com/charles-001/dolphie/blob/main/dolphie/DataTypes.py#L33)

**Comment by charles-001** [comment]
That is very strange... you sure are hitting bugs I've never heard of, lol. I've installed Dolphie on many servers and have never hit this. What is replica_lag value when this happened?

**Comment by charles-001** [comment]
Regardless, I'll fix it with this:
```python
if replica_lag is not None:
        if data.get("SQL_Delay"):
            if isinstance(data["SQL_Delay"], int):
                replica_lag -= data["SQL_Delay"]
```

**Comment by charles-001** [comment]
This is fixed in version `6.6.0`
